### PR TITLE
feat: "a" tag since NIP01 update

### PR DESCRIPTION
**Pull request by bigOconstant**
There is a new tag, called a tag.

### CODE REVIEW

**Comment by ethicnology** [comment]
Hello, thanks for this quick update as they changed the [NIP01 some hours ago](https://github.com/nostr-protocol/nips/commit/e50bf508d9014cfb19bfa8a5c4ec88dc4788d490).

I try to maintain a high percentage of code coverage in this library so if you want this to be merged you should complete the job with a unit test of this new attribute

**Review by ethicnology — CHANGES_REQUESTED** [review]
Unit test is missing

**Comment by ethicnology** [comment]
As you noticed in #34 you may implement this new tag in filter and the matching unit test

**Comment by bigOconstant** [comment]
> As you noticed in #34 you may implement this new tag in filter and the matching unit test

I added the a tag to the filter_test.dart and requests_test.dart unit test which seemed to fit. Let me know if that is enough.

File: test/filter_test.dart [hunk 1]
```diff
@@ -32,6 +34,7 @@ void main() {
       expect(filter.authors, authors);
       expect(filter.kinds, kinds);
       expect(filter.e, e);
+      expect(filter.e, e);
```
Comment (ethicnology) [inline test/filter_test.dart:37]: filter.a, a

Comment (ethicnology) [reply test/filter_test.dart:37]: i will fix it myself

**Review by ethicnology — APPROVED** [review]
Some rework that i will perform

File: test/filter_test.dart (hunk 1 above)
Comment (bigOconstant) [reply test/filter_test.dart:37]: Oops thank you

**Pull request merged**